assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -594,6 +594,135 @@
+		--color-accent-300: #{$muriel-hot-blue-300};
+		--color-accent-300-rgb: #{hex-to-rgb( $muriel-hot-blue-300 )};
+		--color-accent-400: #{$muriel-hot-blue-400};
+		--color-accent-400-rgb: #{hex-to-rgb( $muriel-hot-blue-400 )};
+		--color-accent-500: #{$muriel-hot-blue-500};
+		--color-accent-500-rgb: #{hex-to-rgb( $muriel-hot-blue-500 )};
+		--color-accent-600: #{$muriel-hot-blue-600};
+		--color-accent-600-rgb: #{hex-to-rgb( $muriel-hot-blue-600 )};
+		--color-accent-700: #{$muriel-hot-blue-700};
+		--color-accent-700-rgb: #{hex-to-rgb( $muriel-hot-blue-700 )};
+		--color-accent-800: #{$muriel-hot-blue-800};
+		--color-accent-800-rgb: #{hex-to-rgb( $muriel-hot-blue-800 )};
+		--color-accent-900: #{$muriel-hot-blue-900};
+		--color-accent-900-rgb: #{hex-to-rgb( $muriel-hot-blue-900 )};
+
+		--color-text: #{$muriel-gray-900};
+		--color-text-subtle: #{$muriel-gray-600};
+		--color-surface: #{$muriel-white};
+		--color-surface-backdrop: #{$muriel-gray-0};
+
+		--color-link: #{$muriel-celadon-500};
+		--color-link-rgb: #{hex-to-rgb( $muriel-celadon-500 )};
+		--color-link-dark: #{$muriel-celadon-700};
+		--color-link-dark-rgb: #{hex-to-rgb( $muriel-celadon-700 )};
+		--color-link-light: #{$muriel-celadon-300};
+		--color-link-light-rgb: #{hex-to-rgb( $muriel-celadon-300 )};
+		--color-link-0: #{$muriel-celadon-0};
+		--color-link-0-rgb: #{hex-to-rgb( $muriel-celadon-0 )};
+		--color-link-50: #{$muriel-celadon-50};
+		--color-link-50-rgb: #{hex-to-rgb( $muriel-celadon-50 )};
+		--color-link-100: #{$muriel-celadon-100};
+		--color-link-100-rgb: #{hex-to-rgb( $muriel-celadon-100 )};
+		--color-link-200: #{$muriel-celadon-200};
+		--color-link-200-rgb: #{hex-to-rgb( $muriel-celadon-200 )};
+		--color-link-300: #{$muriel-celadon-300};
+		--color-link-300-rgb: #{hex-to-rgb( $muriel-celadon-300 )};
+		--color-link-400: #{$muriel-celadon-400};
+		--color-link-400-rgb: #{hex-to-rgb( $muriel-celadon-400 )};
+		--color-link-500: #{$muriel-celadon-500};
+		--color-link-500-rgb: #{hex-to-rgb( $muriel-celadon-500 )};
+		--color-link-600: #{$muriel-celadon-600};
+		--color-link-600-rgb: #{hex-to-rgb( $muriel-celadon-600 )};
+		--color-link-700: #{$muriel-celadon-700};
+		--color-link-700-rgb: #{hex-to-rgb( $muriel-celadon-700 )};
+		--color-link-800: #{$muriel-celadon-800};
+		--color-link-800-rgb: #{hex-to-rgb( $muriel-celadon-800 )};
+		--color-link-900: #{$muriel-celadon-900};
+		--color-link-900-rgb: #{hex-to-rgb( $muriel-celadon-900 )};
+
+		--color-button-primary-background-hover: #{$muriel-hot-blue-400};
+		--color-button-primary-scary-background-hover: #{$muriel-hot-red-400};
+
+		--masterbar-color: #{$muriel-white};
+		--masterbar-border-color: #{$muriel-celadon-700};
+		--masterbar-item-new-editor-background: #{$muriel-gray-500};
+		--masterbar-item-new-editor-hover-background: #{$muriel-gray-600};
+		--masterbar-toggle-drafts-editor-background: #{$muriel-gray-600};
+		--masterbar-toggle-drafts-editor-border-color: #{$gray-lighten-20};
```

Comment:
we're trying to drop all the old color variables. Could you please use a `$muriel-gray-???` here instead?